project-2/Handling_Missing_Data.ipynb

```diff
@@ -0,0 +1,8520 @@
+{
```

Comment:
<p>Avoid dropping data "inplace". Leave a backup of the original data so you always go back to reference it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/Humphery_Ufuoma/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>